### ISSUE TITLE
Intellisense for AutoScaleSettings recommends "statistics" when it should be "statistic"

### ISSUE THREAD

**Issue by alan-andrews** [microsoft/vscode-azurearmtools#1141]
The Intellisense for ARM Tools is recommending an illegal property name.  This results in deployment failures that is difficult to debug.

This link describes the issue that I ran into...

https://social.msdn.microsoft.com/Forums/en-US/428ea1a1-1884-4921-96a2-3b7c8a72b2c1/azure-service-plan-configure-autoscale-settings-in-arm-template

In short, I was using intellisense to fill out property names, and it recommended "statistics" (json path properties/profiles[0]/rules[0]/metricTrigger/statistics).

If you compare this with official documentation, it should be "statistic"...

https://docs.microsoft.com/en-us/azure/templates/microsoft.insights/autoscalesettings
(see MetricTrigger object).

Also note that this page shows only 2015-04-01 as the API version where Intellisense only recognizes 2014-04-01

**Comment by StephenWeatherford** [comment]
@anthony-c-martin

**Comment by anthony-c-martin** [comment]
> Also note that this page shows only 2015-04-01 as the API version where Intellisense only recognizes 2014-04-01

This is being tracked by https://github.com/Azure/azure-resource-manager-schemas/issues/843.

**Comment by StephenWeatherford** [comment]
```
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {},
    "functions": [],
    "variables": {},
    "resources": [
        {
            "name": "string",
            "type": "microsoft.insights/autoscalesettings",
            "apiVersion": "2015-04-01",
            "location": "string",
            "tags": {},
            "properties": {
                "profiles": [
                    {
                        "name": "string",
                        "capacity": {
                            "minimum": "string",
                            "maximum": "string",
                            "default": "string"
                        },
                        "rules": [
                            {
                                "metricTrigger": {
                                    "metricName": "string",
                                    "metricResourceUri": "string",
                                    "timeGrain": "string",
                                    "statistic": "string",
                                    "timeWindow": "string",
                                    "timeAggregation": "string",
                                    "operator": "string",
                                    "threshold": "number"
                                },
                                "scaleAction": {
                                    "direction": "string",
                                    "type": "string",
                                    "value": "string",
                                    "cooldown": "string"
                                }
                            }
                        ],
                        "fixedDate": {
                            "timeZone": "string",
                            "start": "string",
                            "end": "string"
                        },
                        "recurrence": {
                            "frequency": "string",
                            "schedule": {
                                "timeZone": "string",
                                "days": [
                                    "string"
                                ],
                                "hours": [
                                    "integer"
                                ],
                                "minutes": [
                                    "integer"
                                ]
                            }
                        }
                    }
                ],
                "notifications": [
                    {
                        "operation": "Scale",
                        "email": {
                            "sendToSubscriptionAdministrator": "boolean",
                            "sendToSubscriptionCoAdministrators": "boolean",
                            "customEmails": [
                                "string"
                            ]
                        },
                        "webhooks": [
                            {
                                "serviceUri": "string",
                                "properties": {}
                            }
                        ]
                    }
                ],
                "enabled": "boolean",
                "name": "string",
                "targetResourceUri": "string"
            }
        }
    ],
    "outputs": {}
}
```

**Comment by StephenWeatherford** [comment]
@alan-andrews This I believe should have been fixed in the 14.1 release (about a month ago).  Please let us know if you're still having problems, thanks!

**Comment by alan-andrews** [comment]
@StephenWeatherford , I just confirmed that this is working for me.  Thanks again!
